### PR TITLE
feat: #717 enable to set a maximum number of aggregations

### PR DESCRIPTION
**Pull request by adipaul1981**

### CODE REVIEW

File: modules/middleware/src/buildAggregations/createFieldAggregation.js [hunk 1]
```diff
@@ -25,6 +25,9 @@ const createNumericAggregation = ({ type, field, graphqlField }) => {
 };
 
 const createTermAggregation = ({ field, isNested, graphqlField }) => {
+  const maxAggregations = get(graphqlField, ['buckets', '__arguments', 0], {
+    max: { value: MAX_AGGREGATION_SIZE },
+  }).max.value;
```
Comment (caravinci) [inline modules/middleware/src/buildAggregations/createFieldAggregation.js:30]: We can also `get` the value directly using the path array, making it easier to read.

```suggestion
  const maxAggregations = get(
    graphqlField,
    ['buckets', '__arguments', 0, 'max', 'value'],
    MAX_AGGREGATION_SIZE
  );
```

**Review by caravinci — CHANGES_REQUESTED** [review]
Thanks for the contribution!
Just a tiny request/suggestion, to make life easier for future devs.

File: modules/middleware/src/buildAggregations/createFieldAggregation.js (hunk 1 above)
Comment (adipaul1981) [reply modules/middleware/src/buildAggregations/createFieldAggregation.js]: done

**Review by caravinci — APPROVED** [review]
LGTM :shipit:

**Pull request merged**